### ISSUE TITLE
Items loose their aspects

### ISSUE THREAD

**Issue by VimiZap** [Azanor/thaumcraft#1730]
I have started with thaumcraft, and I discovered something weird when I was going to make some niter for the very first time in this world. Neither coal nor charcoal had any aspects on them, and when I threw them in the crucible, nothing happend! Earlier when I first scanned these items they had aspects on them, but now they don't and I can't scan the again. I am not sure if there are other items who doesn't have any aspects on them when they should, but there might be some.

**Comment by scepile3** [comment]
I'm running into the same issue; I can't scan coal, charcoal, and sand. After testing with different combinations of mods I use, I think that Extra Utilities may be related to the issue.

With a new instance of Forge (v10.13.3.1360), and Thaumcraft (v4.2.3.5), I was able to scan coal, charcoal, and sand. When I add Extra Utilities 1.23, I can still scan the aspects.
However, when I upgrade to Extra Utilities 1.24, I can't scan coal, charcoal, or sand. The same happens with Extra Utilities 1.24a

I tested using the following procedure:
1. I created a new instance of Minecraft (v1.7.10) with MultiMC.
2. Using the "Install Forge" button in MultiMC, I installed Forge (v10.13.3.1360).
3. Run the game once, and exit...
4. In the "Loader mods" section of MultiMC, I included Thaumcraft (v4.2.3.5).
	When I tested with Extra Utilities, I would add it here also.
I tried with Extra Utilities v1.23 )
5. I ran Minecraft, and created a new world. Game type = creative.
6. Using the creative search menu, I gave myself: a Thaumometer, coal, charcoal, and sand.
7. I tried scanning coal, charcoal, and sand.
